### PR TITLE
CLI changes for klatyn, celo and moonbeam

### PR DESCRIPTION
**Pull request by bruce-riley**
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/1159)

### CODE REVIEW

**Comment by bruce-riley** [comment]
This PR will be submitted after PR 1150 is merged and a new SDK version is released.

File: clients/js/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "wormhole-client",
   "version": "1.0.0",
   "dependencies": {
-    "@certusone/wormhole-sdk": "^0.3.0",
```
Comment (kcsongor) [inline clients/js/package.json:5]: how does this work if the sdk is not a dependency? 🤔

Comment (bruce-riley) [reply clients/js/package.json:5]: Dang it. I didn't mean to delete that line. I meant to delete the line that was pointing at the local version. Putting this back.

**Review by panoel — APPROVED** [review]

File: clients/js/package-lock.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
       "name": "wormhole-client",
       "version": "1.0.0",
       "dependencies": {
-        "@certusone/wormhole-sdk": "^0.3.0",
```
Comment (evan-gray) [inline clients/js/package-lock.json:11]: I think you missed recommitting the package-lock in the last commit

**Review by evan-gray — CHANGES_REQUESTED** [review]

File: clients/js/package-lock.json (hunk 2 above)
Comment (bruce-riley) [reply clients/js/package-lock.json:11]: Fixed.

**Pull request merged**